app/models/job_execution.rb

```diff
@@ -76,6 +76,16 @@ def on_complete(&block)
     @subscribers << JobExecutionSubscriber.new(job, block)
   end
 
+  def self.start_pending_deploys
+    Job.non_deploy.pending.each do |job|
+      JobExecution.start_job(JobExecution.new(job.commit, job))
+    end
+
+    Deploy.active.each do |deploy|
+      deploy.pending_start! if deploy.pending_non_production?
+    end
+  end
```

Comment:
do you suggest it all in application.rb or job things here and deploy things in deploy ?